src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -143,6 +147,42 @@ struct DisallowedParamsInserter {
   mobile_apis::VehicleDataResultCode::eType code_;
 };
 
+ResponseInfo::ResponseInfo()
```

Comment:
@LuxoftAKutsan 
Maybe I put wrong reference so here is the right one https://github.com/smartdevicelink/sdl_core/pull/1640/files#diff-40fc561d5e6483e7ac8f2430329a4674R49
Application manager reference is passed to constructor which is useless as there is same member in the class.